native/ios/README.md

```diff
@@ -15,13 +15,6 @@ It is **strongly advised** to use the provided WebRTC framework and not
 replace it with any other build, since the provided one is known to work
 with the SDK.
 
-## Required Files - Git LFS
-
-PLEASE INSTALL GIT LFS FROM THIS LOCATION - https://git-lfs.github.com/
-
-After Git LFS installation from above link, in your project directory run this command - git lfs install 
-(Note: You should above command before `pod install` command)
-
 ## Using the SDK
```

Comment:
@chepury-bharathraj  - Documentation got removed.

Reply:
Updated The doc.